<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to Edaid (Duplicate Image Finder)</string>
  </property>
  <property name="windowIcon">
   <iconset theme="../resources/icons/Icon.ico">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Welcome_Label">
      <property name="text">
       <string>Edaid:
 Duplikat-Finder für Fotos/Bilder</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_new_project_button">
      <property name="text">
       <string>Neues Projekt Erstellen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="open_existing_project_button">
      <property name="text">
       <string>Gespeichertes Projekt Öffnen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit_button">
      <property name="text">
       <string>Verlassen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
